app/controllers/profiles_controller.rb

```diff
@@ -57,24 +56,20 @@ def dashboard
       calculate_avg_rating
 
       @passengers = Passenger.where(journey_id: Journey.where(car: @car))
-
-      end
```

Comment:
This moved `end` (it's below now) might change meaning. Or maybe git generated a strange diff (happens sometimes)?